Hestia/Hestia/DevicesScreen/AddDeviceScreen/PropertyCell.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UIKit;
+using Foundation;
+using CoreGraphics;
+
+
+namespace Hestia.DevicesScreen.AddDeviceScreen
+{
+    public class PropertyCell : UITableViewCell
+    {
+        public UITextField inputField;
+        public PropertyCell(NSString cellId) : base(UITableViewCellStyle.Default, cellId)
+        {
+            // Create inputfield and add it to the cell
+            inputField = new UITextField();
+            ContentView.AddSubviews(new UIView[] {inputField});
+
+        }
+
+        // Sets the placeholder from the required info
+        public void UpdateCell(string placeholder)
+        {
+            inputField.Placeholder = placeholder;
+                       
+        }
+
+        public override void LayoutSubviews()
+        {
+           base.LayoutSubviews();
+           inputField.Frame = new CGRect(15, 4, ContentView.Bounds.Width - 30, ContentView.Bounds.Height - 8);
```

Comment:
While it should work properly, defining all the exact locations and sizes like this isn't convenient and nice for future changes.